Gopkg.lock

```diff
@@ -130,15 +130,23 @@
   revision = "666e3beca3cb4f6b5c8f176ba80daf2b3adbd84e"
 
 [[projects]]
-  digest = "1:4f66a19975008a3c0b19e3a87737bf140c321f58349ba81d04ec5fd35c8fa773"
+  digest = "1:e06fc49fca4c49d26c2ed3f18fe513b03ff223f759734f0e47f2cf5647215b52"
   name = "github.com/coreos/go-systemd"
   packages = [
     "dbus",
     "unit",
     "util",
   ]
   pruneopts = ""
-  revision = "7b2428fec40033549c68f54e26e89e7ca9a9ce31"
+  revision = "3a5019e750d573b9710fcabf30b8ba1aacec61fa"
+
+[[projects]]
+  digest = "1:b168b5f54103bbb328987b268867f656c4536b70f7b6fec4726a4619b0ebb5f0"
+  name = "github.com/coreos/pkg"
```

Comment:
@howbazaar was talking about bringing in side dependencies recently, maybe send an email to juju-crew to see if anyone has any concerns.

Reply:
Yes, new deps need sign off from the leads before going in.